remote/remote.go

```diff
@@ -17,8 +18,6 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 	"github.com/google/go-containerregistry/pkg/v1/types"
 	"github.com/pkg/errors"
-
-	"github.com/buildpacks/imgutil"
```

Comment:
We group internal imports separately from external imports. Running `make format` will update this. We should really update the PR checks to fail on formatting inconsistencies.